Dockerfile

```diff
@@ -0,0 +1,32 @@
+# syntax = docker/dockerfile:1.2
+
+FROM docker.io/rustlang/rust:nightly-buster as builder
+
+WORKDIR /build
+
+RUN --mount=type=cache,target=/var/cache/apt \
+  apt-get update && \
+  apt-get install -yqq --no-install-recommends \
+  clang \
+  lld \
+  libasound2-dev \
+  libudev-dev
+
+COPY . .
+
+RUN --mount=type=cache,target=~/.cargo/registry \
+    --mount=type=cache,target=/usr/local/cargo/registry \
+    --mount=type=cache,target=/build/target \
+    cargo build --release -p senior_game_server && \
+    cargo install --path /build/senior_game_server --bin senior_game_server --verbose
+
+FROM docker.io/frolvlad/alpine-glibc:alpine-3.13
```

Comment:
Can we add a target to static build the libc and get away with `FROM scratch` here?

Reply:
The best case scenario would be compiling with musl instead of glibc to allow for fully static binaries, however bevy is [currently working towards support for that](https://github.com/bevyengine/bevy/issues/1348). This is a really fresh issue that looks like is going to be tackled soon, so for now I am going to keep this the same and move to the better solution once it is integrated into the bevy mainline.